components/Dropdown/Dropdown.jsx

```diff
@@ -325,14 +326,31 @@ const Dropdown = ({
 
   const [_id] = useState(id || ID_GENERATOR.next().value);
 
-  const inputFilter = value =>
-    items.filter(
-      item =>
-        _getValue(item)
-          .toLowerCase()
-          .indexOf(value.toLowerCase()) > -1,
+  const normalizeChars = value =>
```

Comment:
Not a block, but would be nice to have this function isolated so we can reuse it on other components.

Suggestion: `Components/shared/` folder. 

Just like `hexToRgba`:

https://github.com/catho/quantum/blob/master/components/shared/hexToRgba.js

Reply:
great idea!
Done!

Reply:
:tada: